gpt_index/readers/database.py

```diff
@@ -0,0 +1,67 @@
+"""Database Reader."""
+
+from typing import List, Optional
+from sqlalchemy import create_engine, text
+
+from gpt_index.readers.schema.base import Document
+from gpt_index.langchain_helpers.sql_wrapper import SQLDatabase
+
+class DatabaseReader(SQLDatabase):
```

Comment:
oh my original thought was to just pass in the sql_database object as a variable (similar to how we pass in sql_database to the structured index), so the user has to initialize a SQLDatabase object on their own.

I thought about it for a bit, I actually don't mind your current approach too much, but instead I would prefer my proposal for a few reasons: 
- here you're overriding the __init__ signature of SQLDatabase, so the old init signature can't be used
- you should also be inheriting from BaseReader, here you're inheriting from SQLDatabase 
- it's consistent with how users create a structured index atm

With my proposal, `load_data` would do a similar call to sql_database to run the query, return the result, and would then convert them to Document objects